src/loader_native.ts

```diff
@@ -120,21 +122,16 @@ export class NativeLoader implements Loader {
 
     // check if the package is already linked, if so, return the link and skip
     // a bunch of work
-    try {
-      await Deno.stat(linkDir);
-      this.#linkDirCache.set(npmPackageId, linkDir);
```

Comment:
I'm not too familiar with the code, but to me it seems like this line is used elsewhere to skip work.

Reply:
Evaluating this function is what gets skipped, as I understand it. This means we can do away with this call.

If you'd rather I kept the behaviour as-is, happy to oblige!

Reply:
Done, I’ve left the cache untouched!